cmd/controller/ctrl.go

```diff
@@ -78,6 +81,13 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	if err != nil {
 		glog.Error("Unable to push disk objects to etcd : ", err)
 	}
+	for true {
```

Comment:
1/ Can we avoid this infinite loop?

2/ Can we provide some comments about this loop.

Reply:
comments added